### PR TITLE
[IMP] l10n_mx : Update master data with new Exento tax and fix the XM…

### PR DESCRIPTION
**Pull request by igbe-odoo**
Add a new tax, Exento, to the master data of the module l10n_mx, change the fiscal position 'Foreign Customer' and add a new tax group for Exento.

Add the Exento tax recognition for the XML reader for vendor bills.

task id : 3774996

### CODE REVIEW

File: addons/l10n_mx/data/template/account.tax-mx.csv [hunk 1]
```diff
@@ -75,19 +75,27 @@
 "","","","","","","","","","","","","","","tax","invoice","cuenta216_10_20","-DIOT: Retención","",""
 "","","","","","","","","","","","","","","base","refund","","","",""
 "","","","","","","","","","","","","","","tax","refund","cuenta216_10_20","+DIOT: Retención","",""
-"tax13","19","0%","","VAT(0%)","0.0","percent","purchase","tax_group_iva_0","on_payment","cuenta119_01","","Tasa","","base","invoice","","+DIOT: 0%","","IVA(0%)"
```
Comment (malb-odoo) [inline addons/l10n_mx/data/template/account.tax-mx.csv:78]: Maybe we could leave the sequence like they were to keep the github history :smile:

**Review by malb-odoo — CHANGES_REQUESTED** [review]
Just minor change

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**